tensilelite/rocisa/rocisa/src/code.cpp

```diff
@@ -166,7 +171,12 @@ void init_code(nb::module_ m)
         .def("count", &rocisa::Module::count)
         .def("getItem", &rocisa::Module::getItem)
         .def("setItem", &rocisa::Module::setItem)
-        .def("setItems", &rocisa::Module::setItems)
+        // nanobind cannot reliably bind the same method signatures that differ
+        // only by const-ness, so indirectly bind via a lambda.
+        .def("setItems", [](rocisa::Module& self, 
```

Comment:
Isn't this method signature unique?

Reply:
Not in the Microsoft abi. It's a rather esoteric corner case interaction between the template metaprogramming nanobind is doing and some constraints on the Microsoft side. My suggestion here would be to just accept it and move on. The full explanation requires language lawyering c++ and some of the higher order method pointer strength reduction that nanobind does...

Reply:
Can you add a comment that this is a Windows only issue?